website_sale_product_attribute_filter_visibility/tests/test_attribute_filter_visibility.py

```diff
@@ -6,13 +6,13 @@
 class WebsiteSaleHttpCase(HttpCase):
 
     def test_ui_website(self):
-        """Test frontend tour."""
+        """Test tour."""
         tour = (
             "odoo.__DEBUG__.services['web_tour.tour']",
             "website_sale_product_attribute_filter_visibility",
         )
-        self.browser_js(
-            url_path="/",
+        self.phantom_js(
```

Comment:
phantmo_js is just an alias to browser_js (chrome headless) now. There's no need to change that

Reply:
thanks @chienandalu , I was not aware that phantom_js is an alias. Now is again browser_js. Already checked in travis and runbot. Please, let me know if you prefer a commit splash for only 1 commit

Reply:
Sure, @cor9003 having all commits squashed ease the review labor a lot :)

Reply:
@chienandalu all in only one commit named [MIG] Migration to 12.0